src/video/ogc/SDL_ogcgxcommon.c

```diff
@@ -40,23 +40,24 @@ static const f32 tex_pos[] __attribute__((aligned(32))) = {
     1.0,
 };
 
-void OGC_set_viewport(int x, int y, int w, int h)
+void OGC_set_viewport(int x, int y, int w, int h, int orthoWidth)
```

Comment:
Mmm... I find this orthoWidth rather difficult to understand. It would be nice to have a comment somewhere in the code that explains what it is -- it seems to me that a better name might be screenWidth or displayWidth (assuming that I understood the meaning correctly!), but I also wonder if it would be simpler if we pass a "`float aspect`" instead, or even (if this is enough to cover all the cases) a "`bool wideScreen`". Passing the width directly here is indeed more efficient, but it comes at the cost of readability, and since this is not a speed-critical path, I'd rather trade a couple of CPU cycles for more clarity.